src/cloudsc_fortran_pyiface/pythonsrc/cloudsc_data.py

```diff
@@ -0,0 +1,382 @@
+"""
+cloudsc_data module consist of utilities that:
+- load variables serving as an input to a Fortran computational kernel;
+- load physical parameters needed by a Fortran kernel;
+- load reference results that will be compared with an output of Fortran computation;
+- validates reference vs. computed fields;
+- other, purely technical utilities.
+"""
+from collections import OrderedDict
+import h5py
+import numpy as np
+import cloudsc as clsc
+
+
+NCLV = 5      # number of microphysics variables
+
+
+def define_fortran_fields(nproma,nlev,nblocks):
```

Comment:
Why not turn this into a class? Instead of storing fields in `locals()` you could attach them to the `CloudscFields` object via `setattr`, if you would like to keep the dynamic introspection loops below? But you'd create these in the constructor and allow the outer layers to access them via 
```
fields = CloudscField(...)
fields.load(...)
fields.prainfrac_toprfz  # This is data object
```